bootstrap.sh

```diff
@@ -13,11 +13,11 @@ project_name=$1
 # install ruby gems
 bundle install
 
-#install node packages
+# install node packages
 yarn
 
-# generate an application.yml from the example file
-cp config/application.yml.example config/application.yml
+# generate a .env from the sample file
+cp .env.sample .env
```

Comment:
Isn't it `.env.development`?

Reply:
I usually just use `.env` for development (some credentials might be secret) and I don't want to commit them. And I would use specific env files (like `.env.test`) when I want to always override the values for some reason (like making the tests vars the same for everyone and also for the CI).

https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use